include/deal.II/differentiation/ad/sacado_math.h

```diff
@@ -0,0 +1,96 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2016 - 2017 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE at
+// the top level of the deal.II distribution.
+//
+// ---------------------------------------------------------------------
+
+#ifndef dealii_differentiation_ad_sacado_math_h
+#define dealii_differentiation_ad_sacado_math_h
+
+#include <deal.II/base/config.h>
+
+#ifdef DEAL_II_WITH_TRILINOS
+
+#include <deal.II/base/numbers.h>
+#include <deal.II/differentiation/ad/ad_number_traits.h>
+#include <deal.II/differentiation/ad/sacado_number_types.h>
+
+
+#ifndef DOXYGEN
+
+/**
+ * Define some missing fundamental math functions
```

Comment:
don't you have this namespace documented above?

Reply:
Yes, but since the documentation is within a `#ifndef DOXYGEN` it doesn't get built. Its just an internal / inline comment really. Would you prefer me to change from using `/*...*/` to `//...`?

Reply:
Nah, keep it as-is, I haven’t noticed that

Reply:
actually, I think I'd prefer the `//` to make it clear to a reader that it's not a doxygen comment.

Reply:
@bangerth Do you then want this for all comment in the file, or just the namespace?

Reply:
It's not an important enough issue for you to go back and change stuff. But for things that are not intended to be processed by doxygen, I tend to not use doxygen-style comments to make the intent clear.

Reply:
Ok, thanks for the clarification the leniency. I'll make an effort to change all of these at some point in the future. I've also made a [TODO card](https://github.com/dealii/dealii/projects/7#card-5705772) for this in the AD project on Github.